src/gui/MatchTemplatePanel.cpp

```diff
@@ -491,11 +507,12 @@ void MatchTemplatePanel::SetInputsForPossibleReRun(bool set_up_to_resume_job) {
     SetAndRememberEnableState(DefocusSearchStepNumericCtrl, was_enabled_DefocusSearchStepNumericCtrl, enable_value);
 
     // We still want things that are needed to re-run the job to be enabled.
-#ifdef ENABLEGPU
-    UseGpuCheckBox->SetValue(true);
-#else
-    UseGpuCheckBox->SetValue(false); // Already disabled, but also set to un-ticked for visual consistency.
-#endif
+    // This might not be needed
+    //#ifdef ENABLEGPU
+    //    UseGpuCheckBox->SetValue(true);
```

Comment:
Have we recorded anywhere if this value was set true on a previous run? Does it matter?

Reply:
Playing with it I don't think it matters. This box is never disabled and the value should not be changed. This is always up to the user depending on which compute resources are available at the moment.

We should think at some point about whether this should be set in the run profile instead.